locked-asset/simple-lock/src/proxy_lp.rs

```diff
@@ -0,0 +1,290 @@
+elrond_wasm::imports!();
+elrond_wasm::derive_imports!();
+
+use crate::locked_token::{LockedTokenAttributes, PreviousStatusFlag, UnlockedPaymentWrapper};
+
+#[derive(TypeAbi, TopEncode, TopDecode, NestedEncode, NestedDecode, PartialEq, Debug)]
+pub struct LpProxyTokenAttributes<M: ManagedTypeApi> {
+    pub lp_address: ManagedAddress<M>,
+    pub lp_token_id: TokenIdentifier<M>,
+    pub first_token_id: TokenIdentifier<M>,
+    pub first_token_locked_nonce: u64,
+    pub second_token_id: TokenIdentifier<M>,
+    pub second_token_locked_nonce: u64,
```

Comment:
nonce == 0 -> fungible / not locked entry ?

Reply:
Yep.